libdnf/hy-iutil.cpp

```diff
@@ -312,22 +314,22 @@ mkcachedir(char *path)
 }
 
 gboolean
-mv(const char *old, const char *new, GError **error)
+mv(const char* oldpath, const char* newpath, GError** error)
 {
-    if (rename(old, new)) {
+    if (rename(oldpath, newpath)) {
```

Comment:
To be consistent with old_str and new_str earlier in this PR, consider renaming to old_path and new_path.